skimage/io/tests/test_pil.py

```diff
@@ -199,10 +199,10 @@ def test_imsave_incorrect_dimension():
     with temporary_file(suffix='.png') as fname:
         with testing.raises(ValueError):
             with expected_warnings([fname + ' is a low contrast image']):
-                imsave(fname, np.zeros((2, 3, 3, 1)))
+                imsave(fname, np.zeros((2, 3, 3, 1)), check_contrast=True)
```

Comment:
I think when you revert the default behaviour, you can avoid specifying `check_contrast` here